target_chains/ton/contracts/contracts/tests/PythTest.fc

```diff
@@ -41,3 +38,19 @@ const int OP_UPDATE_GUARDIAN_SET = 2;
 (int) test_get_update_fee(slice in_msg_body) method_id {
     return get_update_fee(in_msg_body);
 }
+
+(int) test_get_chain_id() method_id {
+    return get_chain_id();
+}
```

Comment:
maybe you explained this before, couldn't get test directly call the underlying function?